### PR TITLE
Add SFG space

### PR DESCRIPTION
**Pull request by leonchan0810**
**Add space for ...**  
  
Etherscan: https://etherscan.io/token/0x8a6ACA71A218301c7081d4e96D64292D3B275ce0
CoinGecko: https://www.coingecko.com/en/coins/s-finance
Website: https://s.finance
Twitter: https://twitter.com/SFinanceEx

### CODE REVIEW

**Review by e-asia — CHANGES_REQUESTED** [review]
image
The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.

File: spaces/domains.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "yearn.snapshot.page": "yearn",
   "signal.aave.com": "aave",
   "pillar.snapshot.page": "pillar",
-  "signal.pickle.finance": "pickle"
+  "signal.pickle.finance": "pickle",
+  "signal.s.finance": "s-finance"
```
[the remark this answers is not in the document]
Comment (leonchan0810) [reply spaces/domains.json:6]: @e-asia I fixed the style problem of the file skin.scss, please review it, thx.
@bonustrack please add this custom domain, thx

**Comment by leonchan0810** [comment]
@e-asia When can pr pass? Our team expect our snapshot page can be online before 1st Oct.
Is there anything I can do to speed up this process? Thanks.

File: spaces/s-finance/skin.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.s-finance {
+  --primary-color: #EB3C00;
+  --bg-color: #F7F5EC;
+  --text-color: #262626;
+  --link-color: #EB3C00;
```
Comment (e-asia) [inline spaces/s-finance/skin.scss:5]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/s-finance/skin.scss (hunk 2 above)
Comment (leonchan0810) [reply spaces/s-finance/skin.scss]: Finally, I delete this file to use default skin.
@e-asia Please check it again. Thanks a lot.

**Review by e-asia — APPROVED** [review]

**Pull request merged**